lib/linguist/heuristics.yml

```
@@ -98,6 +98,12 @@ disambiguations:
   rules:
   - language: FreeBasic
     pattern: '^[ \t]*#(?:define|endif|endmacro|ifn?def|if|include|lang|macro)\s'
+- extensions: ['.bs']
+  rules:
+  - language: Bikeshed
+    pattern: '^(?i:<\/*pre>)|(?i:<pre class=.*>)'
+  - language: BrighterScript
+    pattern: '(?i:\w+\s*\(((?:[^()]*))?\s*\)(.*\n*\r*\s*)*?(end\s)(function|sub))'
```

What's going on here? This pattern makes no sense:

~~~regex
.*\n*\r*\s*
~~~

Did you mean to write `(?:.|\n|\r)*` instead? Note that the `dotall` modifier is switched on by default in Ruby (when in Multiline mode, at least), so `.` will also match a carriage return (`\r`) or line-feed (`\n`).

In fact, this entire bit is superfluous and ~~can~~ _should_ be removed for better performance:

```suggestion
    pattern: '(?i:\w+\s*\(((?:[^()]*))?\s*\)(end\s)(function|sub))'
```

**Full disclosure:** I'm entirely unfamiliar with BrighterScript (or its dimmer predecessor, BrightScript).